lms/static/js/student_account/views/FormView.js

```diff
@@ -197,7 +198,7 @@
                     len = errors.length;
 
                 for ( i=0; i<len; i++ ) {
-                    html.push( errors[i] );
+                    html.push( HtmlUtils.ensureHtml(errors[i]) );
                 }
 
                 $msg.html( html.join('') );
```

Comment:
Since the linter will likely complain about this line, it might be better to move the ensure call here.